README.md

```diff
@@ -201,6 +202,13 @@ The following CMake options are available for adding/removing project features.
 
       i.e. cmake -DTRANSPORT_CURL=ON ..
 
+### Consume SDK for C as Dependency with CMake
+Azure SDK for C can be automatically checked out by cmake and become a build dependency. This is done by using [FetchContent](https://cmake.org/cmake/help/v3.11/module/FetchContent.html). 
+
+Using this option would skip manually getting the Azure SDK for C source code to build and installing it (or making it available from some include path). Instead, CMake would do this for us.
+
+Azure SDK for C provides a [CMake module](https://github.com/Azure/azure-sdk-for-c/blob/master/cmake-modules/AddAzureSDK.cmake) that can be copied and used for this purpose.

```

Comment:
From what I understand, `Link verification` is expected to fail on links to new files added in the repo, since they point to master and the file doesn't exist. For such changes, we need to make them in two steps:
1) Check-in the file and doc, but do not add any link to it.
2) Open a new PR that adds the link once the file has been added to master.

cc @weshaggard, @sima-zhu - in case you have other suggestions to make that workflow better. If we can have a short doc section that explains the recommended workflow, that would be great.

Reply:
Btw, are all these exceptions in the logs for link verification expected (looks like most are about anchor tags in URLs)?
```text
Found 25 links on page file:///D:/a/1/s/CONTRIBUTING.md
Exception while requesting #developer-guide
System.UriFormatException: Invalid URI: The hostname could not be parsed.
   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at System.Uri..ctor(String uriString)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.CheckProtocol(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.PrepareUri(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.GetRequest(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.ProcessRecord()
   at System.Management.Automation.CommandProcessor.ProcessRecord()
Exception while requesting #pull-requests
System.UriFormatException: Invalid URI: The hostname could not be parsed.
   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at System.Uri..ctor(String uriString)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.CheckProtocol(Uri ur
...
...
Exception while requesting #vscode
System.UriFormatException: Invalid URI: The hostname could not be parsed.
   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at System.Uri..ctor(String uriString)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.CheckProtocol(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.PrepareUri(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.GetRequest(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.ProcessRecord()
   at System.Management.Automation.CommandProcessor.ProcessRecord()
Exception while requesting #windows
System.UriFormatException: Invalid URI: The hostname could not be parsed.
   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at System.Uri..ctor(String uriString)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.CheckProtocol(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.PrepareUri(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.GetRequest(Uri uri)
   at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.ProcessRecord()
   at System.Management.Automation.CommandProcessor.ProcessRecord()
[503] while requesting https://code.visualstudio.com/
```